### PR TITLE
Export enum for each protocol

### PR DESCRIPTION
**Pull request by pcowgill**
Resolves OU-2041

### CODE REVIEW

**Review by ponyjackal — APPROVED** [review]

File: src/contextualizers/protocol/frenpet/gameplay.ts [hunk 1]
```diff
@@ -229,7 +231,7 @@ export const generate = (transaction: Transaction): Transaction => {
             minter,
             contextAction: {
               type: 'contextAction',
-              value: 'MINTED',
+              value: FrenpetContextActionEnum.MINTED, // TODO: Make a Zora version of this
```
Comment (pcowgill) [inline src/contextualizers/protocol/frenpet/gameplay.ts:234]:
```suggestion
              value: FrenpetContextActionEnum.MINTED, // TODO: Make a Frenpet version of this
```

File: src/contextualizers/protocol/frenpet/gameplay.ts [hunk 2]
```diff
@@ -9,6 +9,8 @@ import {
   ERC20Asset,
   ContextERC721Type,
   ERC721Asset,
+  HeuristicContextActionEnum,
```
Comment (pcowgill) [inline src/contextualizers/protocol/frenpet/gameplay.ts:12]:
```suggestion
```

File: src/contextualizers/protocol/superchainFaucet/superchainFaucet.ts [hunk 3]
```diff
@@ -41,7 +46,7 @@ export function generate(transaction: Transaction): Transaction {
       },
       received: {
         type: 'contextAction',
-        value: 'RECEIVED',
+        value: HeuristicContextActionEnum.RECEIVED,
```
Comment (pcowgill) [inline src/contextualizers/protocol/superchainFaucet/superchainFaucet.ts:49]:
```suggestion
        value: HeuristicContextActionEnum.RECEIVED, // TODO: Make a Superchain version of this
```

File: src/contextualizers/protocol/uniswapV3/uniswapV3Pair.ts [hunk 4]
```diff
@@ -86,7 +87,7 @@ export const generate = (transaction: Transaction): Transaction => {
       },
       contextAction: {
         type: 'contextAction',
-        value: 'SWAPPED',
+        value: HeuristicContextActionEnum.SWAPPED,
```
Comment (pcowgill) [inline src/contextualizers/protocol/uniswapV3/uniswapV3Pair.ts:90]:
```suggestion
        value: HeuristicContextActionEnum.SWAPPED, // TODO: Make a Uniswap version of this
```

File: src/contextualizers/protocol/zoraCreator/zoraCreator.ts [hunk 5]
```diff
@@ -100,7 +101,10 @@ export const generate = (transaction: Transaction): Transaction => {
         value: price,
         unit: 'wei',
       },
-      minted: { type: 'contextAction', value: 'MINTED' },
+      minted: {
+        type: 'contextAction',
+        value: HeuristicContextActionEnum.MINTED,
```
Comment (pcowgill) [inline src/contextualizers/protocol/zoraCreator/zoraCreator.ts:106]:
```suggestion
        value: HeuristicContextActionEnum.MINTED, // TODO: Make a Zora version of this
```

File: src/types/contextAction/index.ts [hunk 6]
```diff
@@ -1,9 +1,7 @@
 import { HeuristicContextAction } from './heuristicContextAction';
 import { ProtocolContextAction } from './protocolContextAction';
 
-export type ContextAction =
-  | HeuristicContextAction
-  | ProtocolContextAction
-  | 'CALLED';
```
Comment (pcowgill) [inline src/types/contextAction/index.ts:7]: Note: This was moved into the heuristic category

File: src/types/contextAction/index.ts [hunk 7]
```diff
@@ -1,9 +1,7 @@
...
+export type ContextAction = HeuristicContextAction | ProtocolContextAction;
 
-export { HeuristicContextAction, ProtocolContextAction };
+export * from './heuristicContextAction';
+export * from './protocolContextAction';
```
Comment (pcowgill) [inline src/types/contextAction/index.ts:7]: @guscost @cdt-eth These are all exported now

**Pull request merged**